modules/lvgl/Kconfig.memory

```diff
@@ -31,28 +31,13 @@ choice LV_Z_MEMORY_POOL
 
 endchoice
 
-if LV_Z_MEM_POOL_SYS_HEAP
-
-config LV_Z_MEM_POOL_MIN_SIZE
-	int "Minimum memory pool block size"
-	default 16
-	help
-	  Size of the smallest block in the memory pool in bytes
-
-config LV_Z_MEM_POOL_MAX_SIZE
+config LV_Z_MEM_POOL_SIZE
```

Comment:
MIN, MAX and BLOCKS are the traces from the past, https://github.com/zephyrproject-rtos/zephyr/pull/28611/files#diff-17227f3b6ec734be2bf8fdfc83c409931dc8f0e1586d528d6335ae237ad72031. You will probably also want to update prompt and help texts of this Kconfig option.

Reply:
Oh yeah, right missed those.